### DISCUSSION TITLE
Workflow Events

### DISCUSSION THREAD

**Discussion by debellotti** [argoproj/argo-workflows#6122] [Ideas]
I was trying to use the argo workflows events and I noticed that some crucial explanations are missing here. I would like to add:

- A simple WorkflowTemplate bound to the WorkflowEventBinding, to show what is triggered by the curl that send the event
- Some infos about the process that bind the event to the workflow template:
   - template creation
   - event binding apply
   - api call to trigger the workflow template creation

Plus: there is a little mistake in the selector: `metadata["x-argo"]` instead of `metadata["X-Argo-E2E"]` I would like to correct it in order to avoid mistakes during the curl.

Hope this is appreciated! ;)

There is a PR about that: https://github.com/argoproj/argo-workflows/pull/6119

Denis

**Comment by agilgur5** [comment]
See the issue & PR that OP made: #6123, #6119
